book/src/internal/build.md

```diff
@@ -0,0 +1,87 @@
+<!--
+SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+
+SPDX-License-Identifier: MIT OR Apache-2.0
+-->
+
+# Build
+
+The process of building a CXX-Qt project internally is complex and contains multiple stages.
+
+  1. CMake build is started, and triggers cargo build
+  2. CXX-Qt build generates C++ sources and static library
+  3. CMake builds the project C++ sources combined with generated sources
+  4. CMake links the target to the Rust static library and Qt
+
+## 1. Cargo build
+
+When cargo build is triggered, this causes dependencies to be downloaded and built. One of these is
+`cxx-qt-lib` which defines helper Qt types. It contains a `build.rs` which uses CXX to generate C++
+sources for the Qt types. These are stored into the `OUT_DIR` of the crate.
```

Comment:
``` md
These are stored into the `OUT_DIR` of the crate. 
```
to:
``` md
These are stored into a single JSON file in the OUT_DIR of the crate.
```

and

``` md
Therefore as `cxx-qt-lib` is built it uses `include_str!`
to expose the generates files as `const` strings in the crate.
```
to:
``` md
Therefore as `cxx-qt-lib` is built it uses `include_str!`
to expose the generated JSON into the QT_TYPES_CXX_JSON variable.
```
I think its helpful to name the variable here, as that makes it a lot easier to find the code where this happens.

Also, we still use "strings" plural in the CXX-Qt build section.

Reply:
Fixed!